src/libs/ReportUtils.js

```diff
@@ -465,27 +465,41 @@ function getIcons(report, personalDetails, policies, defaultIcon = null) {
     return _.map(sortedParticipants, item => item.avatar);
 }
 
+/**
+ * Gets the personal details for a login by looking in the allPersonalDetails from Onyx. If it doesn't exist in Onyx, then a default object is constructed.
```

Comment:
NAB

```suggestion
 * Gets the personal details for a login by looking in the allPersonalDetails Onyx key. If it doesn't exist in Onyx, then a default object is constructed.
```

Reply:
I see where you're going with the suggestion, but `allPersonalDetails` isn't actually an Onyx key. If you want the comment to reference the actual Onyx key, then it should say something like:

> by looking in the ONYXKEYS.PERSONAL_DETAILS Onyx key.

The comment is currently referencing the local variable that the data from the Onyx key is stored in, so I think the comment is correct as originally authored.

Reply:
Ok definitely fair points! I'm still not a huge fan of the way the first sentence is worded, so how about this instead:

```suggestion
 * Gets the personal details for a login by looking in the allPersonalDetails data from Onyx. If it doesn't exist in Onyx, then a default object is constructed.
```

Mainly I don't like `the allPersonalDetails` bit, am I being too picky? Yes, maybe :D

Reply:
OK, I've updated this with a bit of a different suggestion and being slightly more verbose.